src/store/index.ts

```diff
@@ -54,6 +57,15 @@ const {
       }
       return [];
     },
+
+    hasRequiredPermission: (state: State) => (minimumPermission: RoleLevel) => {
+      if (!state.currentWorkspacePermissionInfo) {
+        return false;
+      }
+
+      const { permission } = state.currentWorkspacePermissionInfo;
+      return permission && permission >= minimumPermission;
+    },
```

Comment:
Sure, especially since this is not an intended use of a Vuex getter.

Reply:
I ended up creating a getter to just retrieve the permission level. This seems like the intended use of the getter, and it seems more readable to me than the above replacement, especially since that required some finagling to get typescript to be happy.